src/txmempool.cpp

```diff
@@ -550,6 +550,19 @@ void CTxMemPool::removeConflicts(const CTransaction &tx)
     }
 }
 
+CTxMemPool::~CTxMemPool()
+{
+}
```

Comment:
Why need to add an empty destructor?

Reply:
`CCustomCSView` is forward declared in `txmempool.h` which is incomplete type, so if we did not provide destructor in the cpp we should `include` header with complete type. I prefer to not include but follow forward declaration.